presto-docs/src/main/sphinx/release/release-0.269.rst

```diff
@@ -0,0 +1,33 @@
+=============
+Release 0.269
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Add :func:`uuid` to return a unique identifier.
+* Add JDBC URL parameter doc for MySQL.
+* Add a new configuration property ``spark.memory-revoking-target`` to specify the percent to which memory pool should be occupied after revoke is completed.  This can be overridden by ``spark_memory_revoking_target`` session property.
```

Comment:
This should move under `Presto On Spark Chagnes` section or what ever that section is called.

Reply:
We can just say - 

```
Add a configuration property ``spark.memory-revoking-target`` and session property ``spark_memory_revoking_target`` to specify the percent ...
```

In most situations session properties override configuration properties.

That said "to specify the percent to which memory pool should be occupied after revoke is completed" is vague. Lets frame it in a more understandable terms.